src/main/kotlin/me/serce/solidity/ide/run/SolidityRunState.kt

```diff
@@ -0,0 +1,37 @@
+package me.serce.solidity.ide.run
+
+import com.intellij.execution.application.BaseJavaApplicationCommandLineState
+import com.intellij.execution.configurations.JavaParameters
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.execution.util.JavaParametersUtil
+import com.intellij.openapi.roots.ModuleRootManager
+import com.intellij.util.PathUtil
+import me.serce.solidity.ide.settings.SoliditySettings
+import me.serce.solidity.run.EthereumRunner
+
+
+class SolidityRunState(environment: ExecutionEnvironment?, configuration: SolidityRunConfig) : BaseJavaApplicationCommandLineState<SolidityRunConfig>(environment, configuration) {
+  override fun createJavaParameters(): JavaParameters {
+    val params = JavaParameters()
+    val jreHome = if (myConfiguration.isAlternativeJrePathEnabled) myConfiguration.alternativeJrePath else null
+    params.setJdk(JavaParametersUtil.createProjectJdk(myConfiguration.project, jreHome))
+    setupJavaParameters(params)
+    params.mainClass = EthereumRunner::class.qualifiedName
+    val sourceRoots = ModuleRootManager.getInstance(configuration.configurationModule.module!!).sourceRoots
```

Comment:
> configuration.configurationModule.module!!

This will cause an NPE is no source modules are set up, it's a common case for truffle projects